WordPress/twentytwentyfive
# Replace images of people

**Description**

Closes https://github.com/WordPress/twentytwentyfive/issues/360

Replaces team photo 3 (miner) with person typing on a typewriter.
Replaces the photo of the sugarcane field worker with the nurse.
Replace team photo 1 with woman splashing water.
Replaces cv-bio with woman splashing water.
Replaces team photo 4 with man in hat.

**Testing Instructions**
Test these patterns and confirm that there are no block validation errors:
CV/bio
Link in bio with profile, links and wide margins
Services, team photos
Review with large image on right

Confirm that the old images are deleted.
Confirm that the update to the readme file is correct.

Note: I did not change the aspect ratio of the cv/bio pattern, so it may look slightly different than before.
